src/views/dialogs/DisplaySettingsDialog.tsx

```diff
@@ -44,36 +50,49 @@ export const DisplaySettingsDialog = ({ setIsOpen }: ElementProps) => {
       <Styled.AppThemeRoot value={currentTheme}>
         {[
           {
-            theme: AppTheme.Classic,
+            themeSetting: AppTheme.Classic,
             label: STRING_KEYS.CLASSIC_DARK,
           },
           {
-            theme: AppTheme.Dark,
+            themeSetting: AppThemeSystemSetting.System,
+            label: STRING_KEYS.SYSTEM,
+          },
+          {
+            themeSetting: AppTheme.Dark,
             label: STRING_KEYS.DARK,
           },
           {
-            theme: AppTheme.Light,
+            themeSetting: AppTheme.Light,
             label: STRING_KEYS.LIGHT,
           },
-        ].map(({ theme, label }) => (
-          <Styled.AppThemeItem
-            key={theme}
-            value={theme}
-            backgroundcolor={Themes[theme][currentColorMode].layer2}
-            gridcolor={Themes[theme][currentColorMode].borderDefault}
-            onClick={() => {
-              dispatch(setAppTheme(theme));
-            }}
-          >
-            <Styled.AppThemeHeader textcolor={Themes[theme][currentColorMode].textPrimary}>
-              {stringGetter({ key: label })}
-            </Styled.AppThemeHeader>
-            <Styled.Image src="/chart-bars.svg" />
-            <Styled.CheckIndicator>
-              <Styled.CheckIcon iconName={IconName.Check} />
-            </Styled.CheckIndicator>
-          </Styled.AppThemeItem>
-        ))}
+        ].map(({ themeSetting, label }) => {
+          const theme =
+            themeSetting === AppThemeSystemSetting.System ? AppTheme.Dark : themeSetting;
+
+          const backgroundColor = Themes[theme][currentColorMode].layer2;
+          const gridColor = Themes[theme][currentColorMode].borderDefault;
+          const textColor = Themes[theme][currentColorMode].textPrimary;
+
+          return (
+            <Styled.AppThemeItem
+              key={themeSetting}
+              value={themeSetting}
+              backgroundcolor={backgroundColor}
+              gridcolor={gridColor}
+              onClick={() => {
+                dispatch(setAppThemeSetting(themeSetting));
+              }}
+            >
+              <Styled.AppThemeHeader textcolor={textColor}>
+                {stringGetter({ key: label })}
+              </Styled.AppThemeHeader>
+              <Styled.Image src="/chart-bars.svg" />
+              <Styled.CheckIndicator>
+                <Styled.CheckIcon iconName={IconName.Check} />
+              </Styled.CheckIndicator>
+            </Styled.AppThemeItem>
+          );
```

Comment:
I spent an embarrassing amount of time trying to stack gradients/svgs/backgrounds to get to this design: 
<img width="202" alt="Screenshot 2024-01-31 at 4 05 48 PM" src="https://github.com/dydxprotocol/v4-web/assets/70078372/f0dda91f-bb0f-47db-b76f-904ff6830181">
and still failed (so right now it just looks like dark mode). Would anyone have an idea of the best way to do this?

Reply:
nit/random idea: maybe we can have the color reflect what the current system setting theme is.